client/src/components/DatasetList.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <v-list
+    outlined
+    height="100%"
+  >
+    <v-card
+      v-for="dataset in datasets"
+      :key="dataset.id"
+    >
+      <v-list-item @click="viewDataset(dataset.id)">
+        <v-list-item-content>
+          <v-list-item-title>{{ dataset.name }}</v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+    </v-card>
+  </v-list>
```

Comment:
It might be good to have some sort of separator, space, etc. to distinguish each dataset in the list. Not a huge deal if we can't make it work though.

Reply:
I added an outline border to each dataset and some vertical margins